cookbooks/bcpc-hadoop/recipes/oozie_pack.rb

```diff
@@ -0,0 +1,100 @@
+#
+# Recipe to include hbase jar files into Oozie
+#
+%w{
+hbase
+}.each do |p|
+  package p do
+    action :install
+  end
+end
+
```

Comment:
@pu239ppy . This can be changed to not use ``.each`` construct. It was initially coded to look at ``hbase-client`` ``hbase-master`` components separately.